app/models/station.rb

```diff
@@ -1,4 +1,3 @@
 class Station < ActiveRecord::Base
-  reverse_geocoded_by :latitude, :longitude, address: :location
-  after_validation :reverse_geocode
```

Comment:
I'm not sure how `reverse_geocoded_by` works, but I'm guessing that—since you're removing it—it adds an `after_validation` callback of its own?